p2p/5.Create_an_ERC-20/README.md

```diff
@@ -0,0 +1,245 @@
+✔ Deploy an ERC-20 on an Polygon testnet
+
+## Introduction in a few lines
+
+### What is an ERC-20 ?
+
+**ERC-20**, which stands for "Ethereum Request for Comments 20," is a **technical standard** used to **create** and **manage** tokens based on the **Ethereum blockchain**. ERC-20 tokens are **digital assets** that can represent **various forms** of value, such as financial assets, loyalty points, digital goods, and more.
+
+### Why it is useful ?
+
+The **primary advantage** of ERC-20 lies in its ease of **adoption** and **interoperability**. This standard **defines a set of rules** and functions that allow developers to **create tokens compatible with the Ethereum network** and existing Ethereum wallets. This means that users can store, transfer, and exchange different types of ERC-20 tokens **using the same wallets and interfaces**.
+
+### What technology is used to do this ?
+
+**Solidity** is a **programming language** specifically **designed for developing smart contracts on the Ethereum blockchain**. It enables developers to **create self-executing, autonomous, and verifiable contracts** that define rules and interactions within a **decentralized environment**. Solidity is based on **JavaScript-like syntax** and offers features such as state management, control structures, events, and calls to other contracts, enabling the **creation of complex and secure solutions** on the Ethereum blockchain. If you've never done Solidity before, take a look at the [Solidity Essentials](./Solidity.md) to understand the basics. You can also look at the [official documentation](https://docs.soliditylang.org/en/v0.8.21/).
+
+## Step 0 - Setup
+
+Please refer to the [SETUP.md](./SETUP.md) file.
+
+## Step 1 - Let's start creating our token
+
+### 📑 **Description**:
+
+In this step, you will put in place the necessary foundations for the creation of an ERC-20.
+
+### 📌 **Tasks**:
+
+- First, delete the `script` folder we will not need it as well as the `Counter.sol` and `Counter.t.sol` files.
+- Create a folder `interface` in the `src` folder.
+- Copy the `IERC20.sol` file which is in the `utils` folder to the `interface` folder, previously created.
+    - This file contains the ERC20 [interface](https://docs.soliditylang.org/en/v0.8.21/contracts.html#interfaces), all the functions we are going to implement.
+- Now, create the `ERC20.sol` file in `src`, it is in this file that we will create our ERC-20.
+- Finally, create the `ERC20` contract [inheriting](https://docs.soliditylang.org/en/v0.8.21/contracts.html#inheritance) the `IERC20` interface.
+
+> ⚠️ Don't forget the import and the header of the file
+
+### 📚 **Documentation**:
+
+- [Interface](https://docs.soliditylang.org/en/v0.8.21/contracts.html#interfaces)
+- [Inheritance](https://docs.soliditylang.org/en/v0.8.21/contracts.html#inheritance)
+
+## Step 2 - Let's go on
+
+### 📑 **Description**:
+
+In this step, you will create the variables, the mappings and the constructor necessary for an ERC-20.
```

Comment:
the description should be more appealing, maybe explain what are those variables and how this will be important for building an ERC-20